migrations/20230109144828_add-friendships-index.down.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX IF NOT EXISTS friendships_address_1;
+
+CREATE INDEX IF NOT EXISTS friendships_address_2;
```

Comment:
These should delete the indexes.

Reply:
The autogenerated comment was backwards, weird, changing it